manywheel/build_common.sh

```diff
@@ -77,9 +77,6 @@ fi
 if [[ -n "$DESIRED_PYTHON" && "$DESIRED_PYTHON" != cp* ]]; then
     python_nodot="$(echo $DESIRED_PYTHON | tr -d m.u)"
     case ${DESIRED_PYTHON} in
-      3.[6-7]*)
-        DESIRED_PYTHON="cp${python_nodot}-cp${python_nodot}m"
-        ;;
       # Should catch 3.8+
```

Comment:
This comment "3.8+" no longer relevant after removal of the previous lines?

Reply:
Done. We don't need to check for python version at all here for now.